### PR TITLE
Dev ralucarbc functionality

### PR DESCRIPTION
**Pull request by RalucaRbc**

### CODE REVIEW

**Comment by blowymind99** [comment]
Great!

**Comment by blowymind99** [comment]
Great 👍

File: web/index.html [hunk 1]
```diff
@@ -39,67 +39,71 @@ <h3 class="section-title">About Me</h3>
...
-        <a class="disc-link__janna">
-          <i class="my-cv-twitter"></i>
-        </a>
-      </c>
-      <c span="1">
-        <a class="disc-link__double-pearl-lusta">
-          <i class="my-cv-youtube"></i>
-        </a>
-      </c>
+      <c span="row" span-tablet="4" span-desktop="8">
```
Comment (iuliancarnaru) [inline web/index.html:47]: I am not up to date with the project. but is a bit wired to see `c` instead of `div` ... I assume `c` stands for `column`

**Review by iuliancarnaru — APPROVED** [review]

**Review by psyhedeliq — APPROVED** [review]
LGTM! 🚀 🤘

File: web/index.html (hunk 1 above)
Comment (RalucaRbc) [reply web/index.html:47]: Yes, `c`  stands for `column` and I think this was Marian's way of doing just the starter version of my-cv-v2 just for the purpose of better explaining the grid layout to the us, the starters 😄; hope I'm not wrong

**Pull request merged**